### PR TITLE
[HSEARCH] 6.2.0.CR1

### PR DESCRIPTION
**Pull request by yrodiere**

### CODE REVIEW

**Comment by yrodiere** [comment]
Should be available soon at https://staging.hibernate.org/search/releases/6.2/

File: _data/projects/search/releases/6.2/6.2.0.CR1.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+date: 2023-06-26
+announcement_url: https://in.relation.to/2023/06/26/hibernate-search-6-2-0-CR1
+summary: >-
+  New `excludePaths` filter for `@IndexedEmbedded`,
```
Comment (marko-bekhta) [inline _data/projects/search/releases/6.2/6.2.0.CR1.yml:4]: hmm, looks like it doesn't want to treat content from this yml as asciidoctor:
image

Comment (yrodiere) [reply _data/projects/search/releases/6.2/6.2.0.CR1.yml:4]: Yes, I'll need to work on that. It's not a new problem though :)

Comment (yrodiere) [reply _data/projects/search/releases/6.2/6.2.0.CR1.yml:4]: I fixed that.

**Pull request merged**